### DISCUSSION TITLE
git clone getting fail

### DISCUSSION THREAD

**Discussion by suchismita1980** [community/community#114664] [Repositories] [answered]
### Select Topic Area

Question

### Body

git clone showing below error:
git clone -b master https://github.com/intel-innersource/os.chrome.chromiumos.platform.tast tast
Cloning into 'tast'...
Username for 'https://github.com': 
Password for '@github.com': 
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/intel-innersource/os.chrome.chromiumos.platform.tast/'

**Comment by nehalp13** [comment, marked as answer]
Hello @suchismita1980 the error message you're encountering indicates that GitHub no longer supports password authentication for cloning repositories over HTTPS. Instead, you should use either SSH authentication or Personal Access Tokens (PATs) for authentication.

Here's how you can resolve this issue:

1. SSH Authentication: If you haven't already set up SSH keys for your GitHub account, you can follow GitHub's documentation to generate SSH keys and add them to your GitHub account. Once configured, you can clone repositories using the SSH URL instead of the HTTPS URL. For example:

git clone -b master git@github.com:intel-innersource/os.chrome.chromiumos.platform.tast tast


2. Personal Access Token (PAT): Alternatively, you can generate a Personal Access Token from your GitHub account settings and use it for authentication. Here's how you can clone the repository using a PAT:

git clone -b master https://username:token@github.com/intel-innersource/os.chrome.chromiumos.platform.tast tast

Replace `username` with your GitHub username and `token` with the Personal Access Token you generated.

3. Credential Manager: You can also use a credential manager to store your GitHub credentials securely. This way, you won't have to provide your username and token or password every time you interact with a GitHub repository. Git provides credential helpers for various platforms.

Once you've chosen one of the above methods and updated your clone command accordingly, you should be able to clone the repository without encountering authentication errors. If you're still having trouble, double-check that you've entered your credentials or token correctly.

**Reply by suchismita1980** [reply]
Hi Nehal, thanks for your help.
After the 2nd step I am getting the below error:
<img width="494" alt="image" src="https://github.com/community/community/assets/48068818/0448d21e-d2e9-4560-8773-1c69d0e6a1b0">

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @suchismita1980! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by suchismita1980** [reply]
Thank you!

**Comment by frankexampazz** [comment]
A critical error occurred in the Clone Source step of the git migration. Failed to clone git source. Please who has experienced this and how did you import successfully?
